drivers/flash/Kconfig.nor

```diff
@@ -5,6 +5,7 @@ DT_COMPAT_JEDEC_SPI_NOR := jedec,spi-nor
 
 menuconfig SPI_NOR
 	bool "SPI NOR Flash"
+	default y
```

Comment:
this property should depends on existence of DT node which describes SPI-NOR memory